packages/base/src/mainview/mainView.tsx

```diff
@@ -732,6 +735,9 @@ export class MainView extends React.Component<IProps, IStates> {
       targetLayerPosition++
     ) {
       const layerId = layerIds[targetLayerPosition];
+      if (this.state.loadingErrors.find(item => item.id === layerId)) {
+        continue;
+      }
```

Comment:
So, if I understand your code correctly: if there is an error just once on this layer, we will never try to create it anymore. Unfortunately not giving the possibility for the user to fix the error.

So I'd like to suggest not doing this:
```suggestion
```